app/js/features/license/saga.js

```diff
@@ -1,13 +1,15 @@
+/*global web3*/
 import License from 'Embark/contracts/License';
 import SNT from 'Embark/contracts/SNT';
 import Escrow from 'Embark/contracts/Escrow';
-import web3 from 'Embark/web3';
 
 import { fork, takeEvery, call, put } from 'redux-saga/effects';
 import {
+  ADD_USER_RATING,
   BUY_LICENSE, BUY_LICENSE_FAILED, BUY_LICENSE_SUCCEEDED,
   CHECK_LICENSE_OWNER, CHECK_LICENSE_OWNER_FAILED, CHECK_LICENSE_OWNER_SUCCEEDED,
-  USER_RATING, USER_RATING_FAILED, USER_RATING_SUCCEEDED
+  USER_RATING, USER_RATING_FAILED, USER_RATING_SUCCEEDED, GET_LICENSE_OWNERS, GET_LICENSE_OWNERS_SUCCCEDED,
+  GET_LICENSE_OWNERS_FAILED
```

Comment:
Do you know why License and Rating are mixed up?

Reply:
Was like that when I started. It kind of makes sense, because your license is like your information also and if you don't have a license, you can't have a rating. 
I guess we can move it out later if we wish

Reply:
Fair enough